src/components/AuthProvidersList.jsx

```diff
@@ -3,7 +3,8 @@ import { AuthProvider } from '@/components/Items';
 import { Box } from '@mui/material';
 import {Google, GitHub, FacebookRounded as Facebook} from '@mui/icons-material';
 
-const availableProviders = [
+
+export const availableProviders = [
```

Comment:
Remove redundant empty line

```suggestion
export const availableProviders = [
```